temporalcli/commandsgen/parse.go

```diff
@@ -44,6 +44,14 @@ type (
 		IgnoreMissingEnv       bool     `yaml:"ignores-missing-env"`
 		Options                []Option `yaml:"options"`
 		OptionSets             []string `yaml:"option-sets"`
+		Docs                   Docs     `yaml:"docs"`
+	}
+
+	// Docs represents docs-only information that is not used in CLI generation.
+	Docs struct {
+		// Tags are the same as Keywords, but with `-` instead of ` `.
+		Keywords    []string `yaml:"keywords-tags"`
+		Description string   `yaml:"description"`
```

Comment:
```suggestion
		DescriptionHeader string   `yaml:"description-header"`
```

I wonder if this makes sense just to differentiate from command description and clarify this is a single-line thing